services/gitdiff/gitdiff.go

```diff
@@ -1327,33 +1328,44 @@ func GetDiff(gitRepo *git.Repository, opts *DiffOptions, files ...string) (*Diff
 	if len(opts.SkipTo) > 0 {
 		argsLength++
 	}
-	if len(files) > 0 {
-		argsLength += len(files) + 1
+	argsLength += len(files)
+
+	// git 1.9.0 added pathspec exclusion of files
```

Comment:
Now the minimal git version is 2.0, so version check is unnecessary.

Reply:
Yep, true.
I'd still like to add some tests before allowing this PR to be merged.
However, I'm still unsure how to do that in this case:
I know there are former git repositories in our codebase that are used for testing somehow, but I have no idea how or where to add tests.

I'll fix it when I add the tests as well.